docs/templates/backend.md

```diff
@@ -108,9 +108,9 @@ Most tensor operations you will need can be done as you would in TensorFlow or T
 
 ```python
 # Initializing Tensors with Random Numbers
-b = K.random_uniform_variable(shape=(3, 4)). # Uniform distribution
-c = K.random_normal_variable(shape=(3, 4)). # Gaussian distribution
-d = K.random_normal_variable(shape=(3, 4)).
+b = K.random_uniform_variable(shape=(3, 4), 0, 1). # Uniform distribution
+c = K.random_normal_variable(shape=(3, 4), 3, 1). # Gaussian distribution
+d = K.random_normal_variable(shape=(3, 4), 3, 1).
```

Comment:
This is incorrect syntax (keyword arg before positional args). Use keyword args everywhere for explicitness.